### PR TITLE
Fix 6lu7 simulation script

### PR DESCRIPTION
**Pull request by zhang-ivy**
Fixes:

- Remove second barostat from simulate_6lu7_complex.py
- Remove .DS_Store files
- Update README

### CODE REVIEW

**Comment by rafwiewiora** [comment]
@zhang-ivy also need to modify the one (or multiple?) `system.xml`'s -- just remove the second barostat line manually.

**Comment by zhang-ivy** [comment]
Just removed the extra barostats from the system.xmls, so this should be ready to be merged now.

**Comment by jchodera** [comment]
Did `SystemGenerator` accidentally add the second barostat? If so, can you file an issue here and I'll make sure to add a check so that doesn't happen again to someone else in the future?
https://github.com/openmm/openmmforcefields/issues

**Comment by zhang-ivy** [comment]
No, I accidentally added it

**Review by jchodera — APPROVED** [review]
Looks good, thanks!

File: system-preparation/6lu7_complex/output/system_4fs.xml [hunk 1]
```diff
@@ -295471,6 +295471,5 @@
 			</Exceptions>
 		</Force>
 		<Force forceGroup="0" frequency="25" pressure="1.01325" randomSeed="110257930" temperature="310" type="MonteCarloBarostat" version="1"/>
```
Comment (jchodera) [inline system-preparation/6lu7_complex/output/system_4fs.xml:295473]: Do we want to change the barostat frequency to 50 for more speed, or are we fine with 25?

Comment (rafwiewiora) [reply system-preparation/6lu7_complex/output/system_4fs.xml:295473]: what's 'more speed'? you have numbers on that?

Comment (rafwiewiora) [reply system-preparation/6lu7_complex/output/system_4fs.xml:295473]: I'll change in the future, obviously not now we have stuff running on F@h

**Pull request merged**